src/plugins/vis_types/timeseries/public/timeseries_vis_renderer.tsx

```diff
@@ -50,6 +50,14 @@ export const getTimeseriesVisRenderer: (deps: {
 
     const showNoResult = !checkIfDataExists(visData, model);
 
+    const initialRender = () => {
+      // We need to wait a little to be sure that all inner components will be fully rendered.
+      // Important for reporting!
+      setTimeout(() => {
+        handlers.done();
+      }, 350);
```

Comment:
Why is this necessary? Doesn't seem like there is a chart type that would require doing this.

Reply:
@flash1293  it's not needed. But I just wanted to double protect that solution. Maybe we can reduce timeout to `100ms`